tools.deps/src/cider/enrich_classpath/clojure.clj

```diff
@@ -162,7 +162,7 @@
                                                                               (when (seq (select-keys m [:git/url :git/sha :git/tag :sha :tag
                                                                                                          :local/root]))
                                                                                 ;; use m (and not the select-keys result) to honor `:exclusions`:
-                                                                                [dep m])))
+                                                                                [dep (assoc m :exclusions classpath-overrides-vector)])))
```

Comment:
The comment in L164 makes be believe `m` can have :exclusions - aren't we at risk of overriding them?

What about:

```
[dep (-> m
         (update :exclusions into classpath-overrides-vector)
         (update :exclusions vec))]
```